src/publisher.ts

```diff
@@ -237,13 +246,17 @@ export class Publisher {
    * @param {object} attributes The message attributes.
    * @param {function} callback The callback function.
    */
-  queue_(data, attrs, callback) {
-    this.inventory_.queued.push({
+  queue_(data: Buffer, attrs: Attributes): Promise<string>;
```

Comment:
Maybe we can keep the attibutes param name consistent (publish has them down as attributes)

Reply:
👍